.github/workflows/test-e2e.yml

```diff
@@ -113,14 +113,11 @@ jobs:
         uses: broadinstitute/datarepo-actions/actions/main@0.67.0
         with:
           actions_subcommand: 'gcp_whitelist_clean'
-  notify-slack:
-    needs: [e2e_test]
-    uses: ./.github/workflows/notify-slack.yaml
-    secrets: inherit
-    if: ${{ always() }}
+  report-workflow:
+    uses: broadinstitute/sherlock/.github/workflows/client-report-workflow.yaml@main
+    if: ${{ github.ref == 'refs/heads/develop' }}
     with:
-      workflow_name: Nightly E2E Tests
-      notify_on_success: true
-      failure_slack_channel_id: "CMYTGJVFY,C53JYBV9A" # #jade-alerts & #dsde-qa
-      success_slack_channel_id: "CK9M0ENRJ,C53JYBV9A" # #jade-spam & #dsde-qa
-      success: ${{ needs.e2e_test.result == 'success' }}
+      notify-slack-channels-upon-workflow-success: "#jade-spam"
```

Comment:
Just curious, is there a reason that you are only notifying #jade-spam on success rather than completion?

Reply:
I found this wording confusing as well - from the docs:

> “Upon completion” means “upon success or failure.” It’s like setting both the “upon success” and “upon failure” fields to the same channel.

Reply:
Right!  That's what I figured it meant, thanks for confirming.

I'd think that we'd want to notify #jade-spam on both success and failure (so on completion), what do you think?

Reply:
Ah, ok - That would be fine. I was just thinking that we send our nightly test results for the api to #jade-alerts, but sending them both to spam would be fine.

Reply:
Probably better to just pick a channel rather than them going to different channels. I can just switch this to #jade-spam.

Reply:
I've updated the channels for [almost all of the alerts:(https://github.com/DataBiosphere/jade-data-repo-ui/pull/1600/commits/87db32250bc67215d50c38b758404feb907c6ce1)
- If we're also sending the alert to QA, then we probably should have the result in jade-alerts. So, e2e is sent to #jade-alerts
- Otherwise, send to jade-spam. I think that's what we were originally doing with our dev image updates. And, I think that's a fine place to report the unit tests.